spec/acceptance/jira/change_issue_status_spec.rb

```diff
@@ -14,13 +14,15 @@
   let(:user_is_developer) { false }
   let(:user_is_reviewer) { false }
   let(:user_is_feature_owner) { false }
+  let(:slack_username) { 'user.name' }
 
   let!(:user) do
     JobTomate::Data::User.create(
       jira_username: jira_username,
       jira_developer: user_is_developer,
       jira_reviewer: user_is_reviewer,
-      jira_feature_owner: user_is_feature_owner
+      jira_feature_owner: user_is_feature_owner,
+      slack_username: slack_username
```

Comment:
Changing the slack user receiving the notification to the user that performed the status change on JIRA